common/lib/api-client/models.test.js

```diff
@@ -52,78 +56,91 @@ const testCases = {
       httpMock: 'get',
       mockPath: '/1',
       args: '1',
+      statusCode: 200,
     },
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   court_hearing: [
     {
       method: 'create',
       httpMock: 'post',
       args: {},
+      statusCode: 200,
     },
   ],
   timetable_entry: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   gender: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   ethnicity: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   assessment_question: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   location: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
     {
       method: 'find',
       httpMock: 'get',
       mockPath: '/1',
       args: '1',
+      statusCode: 200,
     },
   ],
   supplier: [
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
     {
       method: 'find',
       httpMock: 'get',
       mockPath: '/1',
       args: '1',
+      statusCode: 200,
```

Comment:
Maybe have the status code default to `200` unless specificed?

Reply:
I had it like that originally, but thought it was probably better to be explicit. Happy to snip it out though - your call